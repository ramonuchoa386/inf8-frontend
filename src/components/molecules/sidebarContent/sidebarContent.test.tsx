/* eslint-disable @typescript-eslint/no-var-requires */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { rendererCreate } from './../../../utils/test/test-utils';

import SidebarContent from '.';

describe('Componente SidebarContent', () => {
  test('Deve renderizar o menu lateral', () => {
    const sidebar = rendererCreate(
      <MemoryRouter>
        <SidebarContent />
      </MemoryRouter>
    ).toJSON();

    expect(sidebar).toMatchSnapshot();
  });
});
