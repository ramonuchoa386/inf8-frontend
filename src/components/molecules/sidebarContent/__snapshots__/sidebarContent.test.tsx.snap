// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente SidebarContent Deve renderizar o menu lateral 1`] = `
.c0 {
  overflow-y: auto;
  margin-bottom: 20px;
  background: #191918;
}

.c0::-webkit-scrollbar {
  width: 5px;
}

.c0::-webkit-scrollbar-track {
  background: #191918;
}

.c0::-webkit-scrollbar-thumb {
  background: #888;
}

.c0::-webkit-scrollbar-thumb:hover {
  background: #555;
}

.c0 .pro-menu.square {
  padding-top: inherit;
}

.c0 .pro-menu.square .pro-menu-item > .pro-inner-item {
  padding-top: inherit;
  padding-bottom: inherit;
  height: 60px;
}

.c0 .pro-menu.square .pro-menu-item > .pro-inner-item > .pro-icon-wrapper {
  background-color: initial;
}

.c0 nav.pro-menu.inner-submenu-arrows {
  padding-top: 0px;
}

.c1 {
  border: 0px;
  border-top: 1px solid #73736E;
  padding: 0px;
  margin: 0px;
}

<div
  className="pro-sidebar-content c0"
>
  <nav
    className="pro-menu shaped square inner-submenu-arrows"
  >
    <ul>
      <nav
        className="pro-menu inner-submenu-arrows"
        firstchild={1}
        popperarrow={0}
      >
        <ul>
          <li
            className="pro-menu-item"
          >
            <div
              className="pro-inner-item"
              role="button"
              tabIndex={0}
            >
              <span
                className="pro-icon-wrapper"
              >
                <span
                  className="pro-icon"
                >
                  <svg
                    fill="currentColor"
                    height={17}
                    stroke="currentColor"
                    strokeWidth="0"
                    style={
                      Object {
                        "color": undefined,
                      }
                    }
                    viewBox="0 0 24 24"
                    width={17}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M3 13h1v7c0 1.103.897 2 2 2h12c1.103 0 2-.897 2-2v-7h1a1 1 0 0 0 .707-1.707l-9-9a.999.999 0 0 0-1.414 0l-9 9A1 1 0 0 0 3 13zm7 7v-5h4v5h-4zm2-15.586 6 6V15l.001 5H16v-5c0-1.103-.897-2-2-2h-4c-1.103 0-2 .897-2 2v5H6v-9.586l6-6z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="pro-item-content"
              >
                Home
                <a
                  color="#FFF"
                  href="/"
                  onClick={[Function]}
                  title="Dashboard"
                />
              </span>
            </div>
          </li>
          <hr
            className="c1"
          />
        </ul>
      </nav>
    </ul>
  </nav>
</div>
`;
